content/product-engineering/engineering/cloud/growth-and-integrations/index.md

```diff
@@ -0,0 +1,200 @@
+        <div style="text-align: center; font-size: 12px;">(starting 01/10)</div>
+      </div>
+    </div>
+
+<div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/rob-rhyne.png" alt="Picture of Rob Rhyne" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Rob Rhyne</div>
+        <div style="text-align: center; font-size: 12px;">Temporary product manager and product designer, growth</div>
+      </div>
+    </div>
+    <div class="col" style="flex: 1;">
+      <div>
+        <div>
+          <img src="https://storage.googleapis.com/sourcegraph-assets/handbook/growth-and-integrations/join-us-vermillion.png" alt="Picture with CTA: Join Us" style="background: transparent; width:128px;"/>
+        </div>
+        <div style="text-align: center;">Stephen Gutekanst </div>
+        <div style="text-align: center; font-size: 12px;">Software Engineer (growth)</div>
+      </div>
+    </div>
+  </div>
+
+</section>
+
+## Growth vision
+
+_Sourcegraph is optimized to help users discover how powerful the tool is, as quickly as possible._
+
+This vision is necessarily broad because the growth-oriented goal for the team is the generation of Product Qualified Leads. These PQLs will generally be generated from users who find value in the product and integrate Sourcegraph into their daily routines. As such, this team will also impact the following objectives:
+
+- Improve the retention of all users
+- Increase the frequency of users from monthly active to daily active
+- Expand usage in existing accounts
+
+## Integrations vision
+
+Sourcegraph powers your first stop for any information about your code, for any tool or website you use.
+
+- Build a healthy ecosystem of Sourcegraph extensions, built by developers everywhere, who want to bring the power of existing developer tools they love into our platform.
+- Build new paths to get value from Sourcegraph on other websites and IDEs.
+- All of Sourcegraph's most powerful features are easily available on most any code host while using any browser thanks to native integrations and browser extensions.
+
+## Responsibilities and focus
+
+### Growth-oriented responsibilities
+
+In pursuit of goals related to growth, the team will own or contribute to the following:
+
+- Researching how users become activated, sign-up, and generate PQLs
+- Guiding and shaping the initial user journey of:
+  - visitors to sourcegraph.com
+  - authenticated users of Self-hosted installations
+  - TBD: authenticated users to the cloud product
+    - Cloud is working on this, but solutions here probably need to be influenced by user journeys developed for the self-hosted product and vice-versa
+- Creation and optimization of in-product flows that result in:
+  - Cloud sign-ups
+  - Self-hosted installs
```

Comment:
Yes, and this is where it gets confusing for lots of folks. A PQL is for either a Cloud account or a Self-hosted account install. 

At the part of the user journey we are working to improve, we cannot determine which product a given user needs. We can only help them see value and want to bring that value to their team. We will then guide them to a place where they choose the right product for them. 

Because we worked to get the user to either of those actions, it makes sense that we also acknowledge that we will create these events.